packages/middleware/src/errors/defaultErrorHandler.ts

```diff
@@ -1,32 +1,21 @@
-import type { Request } from "express";
 import { getAgnosticStatusCode } from "../helpers";
-import type { AlokaiResponse } from "../types";
-
-type ClientSideError = {
-  message?: string;
-};
+import type { Integration } from "../types";
 
 /**
  * Default error handler for the middleware
- *
- * @param error
- * @param req
- * @param res
  */
-export const defaultErrorHandler = (
-  error: ClientSideError,
-  req: Request,
-  res: AlokaiResponse
+export const defaultErrorHandler: Integration["errorHandler"] = (
+  error,
+  _req,
+  res
```

Comment:
If I try removing the types now in the customer's project, there are still errors because they are implicit type any. 

<img width="1155" alt="image" src="https://github.com/user-attachments/assets/804844e7-6efe-48f0-9f14-37ce224b1c25" />

Reply:
even when you add types for the integration? (`satisfies`)
![CleanShot 2024-12-12 at 18 05 26@2x](https://github.com/user-attachments/assets/128b69d7-3741-4ecb-a8ed-9b94da504e01)

Reply:
that does it, missed that part. Thanks